packages/backend-modules/voting/graphql/schema-types.js

```diff
@@ -260,18 +263,46 @@ type Questionnaire {
 }
 
 input SubmissionsFilterInput {
+  "Return only submission with this ID"
   id: ID
+
+  "Omit submission with this ID"
   not: ID
 
+  "Return only submissions with these IDs"
   submissionIds: [ID!]
+
+  "Omit submissions with these IDs"
   notSubmissionIds: [ID!]
 
-  answeredQuestionIds: [ID!]
-  hasAnswers: Boolean @deprecated(reason: "use \`answeredQuestionIds\` instead")
+  "Return only submissions with these answered question IDs"
+  answeredQuestionIds: [ID!] @deprecated(reason: "use \`answers\` instead")
+
+  "Return only submissions with one or more answers" 
+  hasAnswers: Boolean @deprecated(reason: "use \`answers\` instead")
```

Comment:
Do the deprecated endpoints still work?

Reply:
Yes, deprecated end points still work.

Reply:
As `hasAnswers` is no longer in the must part of the query implementation, how does GraphQl translate this again from the schema to the code if I ask for the `hasAnswers` filter? (I think you explained this already but I can't find it for this example)

Reply:
Deprecation notice is misleading. Since we switched to indexing anonymized submissions, `hasAnswers` has no effect anymore; it will always return submissions with answers – they either belong to users or a pseudonym.